code/go/0chain.net/blobbercore/challenge/challenge.go

```diff
@@ -63,42 +63,48 @@ func syncOpenChallenges(ctx context.Context) {
 
 }
 
-func saveNewChallenge(nextChallenge *ChallengeEntity, ctx context.Context) {
+func saveNewChallenge(c *ChallengeEntity, ctx context.Context) {
 	defer func() {
 		if r := recover(); r != nil {
-			logging.Logger.Error("[recover]challenge", zap.Any("err", r))
+			logging.Logger.Error("[recover]add", zap.Any("err", r))
 		}
 	}()
 
 	db := datastore.GetStore().GetDB()
-	if Exists(db, nextChallenge.ChallengeID) {
+	if Exists(db, c.ChallengeID) {
 		return
 	}
 
 	lastChallengeID, err := getLastChallengeID(db)
 
 	if err != nil {
-		logging.Logger.Error("[challenge]db: ", zap.Error(err))
+		logging.Logger.Error("[challenge]add: ", zap.Error(err))
 		return
 	}
 
-	isValid := nextChallenge.PrevChallengeID == "" || lastChallengeID == nextChallenge.PrevChallengeID
+	isValid := c.PrevChallengeID == "" || lastChallengeID == c.PrevChallengeID
 
 	// it is not First and Next challenge
 	if !isValid {
 		logging.Logger.Error("[challenge]Challenge chain is not valid")
 		return
 	}
 
-	logging.Logger.Info("[challenge]add: ", zap.String("challenge_id", nextChallenge.ChallengeID))
-	nextChallenge.Status = Accepted
-	nextChallenge.CreatedAt = common.ToTime(nextChallenge.Created)
-	nextChallenge.UpdatedAt = nextChallenge.CreatedAt
+	c.Status = Accepted
+	c.CreatedAt = common.ToTime(c.Created)
```

Comment:
@cnlangzi when we hit open_challenges. we respond the creation timestamp as well.. you should check with that as well

Reply:
just logging it first. Because we don't know what is cct on blockchain now. we have to get it from blockchain for checking and rejecting challenge.